policies.md

```diff
@@ -29,30 +43,55 @@ There are a number of ways that policies can be applied directly to influence th
 
 <img src="gcam-figs/policy cost.png" width="750" height="300" />
 
-As discussed in Bradley, et al. (2001) and demonstrated in Calvin, et al. (2014), the approach can be used to calculate costs for a wide range of heterogeneous non-price policies. While conceptually similar to the simple approach above, the other is tedious. Similarly, the deadweight loss approach can be used to calculate the cost of policies other than carbon taxes. It is completely general (Mankiw and Hakes, 2012).
+As discussed in [Bradley et al. (1991)](policies.html#Bradley1991) and demonstrated in [Calvin et al. (2014)](policies.html#Calvin2014), the approach can be used to calculate costs for a wide range of heterogeneous non-price policies. While conceptually similar to the simple approach above, the other is tedious. Similarly, the deadweight loss approach can be used to calculate the cost of policies other than carbon taxes. It is completely general [Mankiw & Hakes (2012)](policies.html#Mankiw2012).
+
 
 The approach is employed at each GCAM time step. Costs occurring between time steps is inferred by interpolation. Costs over time can be summed. Costs can be summed with or without discounting. But, the GCAM user needs to be aware of the implications of whatever approach is employed.
 
 The deadweight loss approach is not without its limitations. While the numerical calculation is simple for a uniform carbon tax (or a cap-and-trade regime), more complex policies are more tedious to represent. Second, there is no link back to the macro-economy. Changes of the magnitude associated with stringent climate policies will have macro-economic consequences. Those consequence will, in turn affect the scale of economic activity. Third, there is no way to calculate the effects of alternative uses of tax revenue or carbon permit allocations.
 
-## References
+## <a name="non-co2-markets"> Markets For non-CO<sub>2</sub> Emission Species</a>
```

Comment:
is this the right place for this?

I'd think this should be next to the ghg-link section.
ideally, first intorduce the concept of non-CO2 market, then explain how to link them.